src/screens/ChatScreen.tsx

```diff
@@ -0,0 +1,478 @@
+              onBackDropPressed={() => handleOnBackDropPressed()}
+            />
+          )}
+        >
+          <NativeViewGestureHandler disallowInterruption={true}>
+            <View style={{ flex: 1 }}>
+              {messageSelected && (
+                <>
+                  <ListItem
+                    onPress={() => handleEditItemPress(messageSelected)}
+                    style={{ paddingLeft: insets.left }}
+                    containerStyle={styles.messageOptionsContainer}
+                    testID={CHAT_MESSAGE_OPTIONS_EDIT}
+                  >
+                    <MaterialIcons name="edit" size={24} color="grey" />
+                    <ListItem.Content style={{ flexDirection: 'row' }}>
+                      <ListItem.Title style={{ flex: 2 }}>
+                        {t('Edit')}
+                      </ListItem.Title>
+                    </ListItem.Content>
+                  </ListItem>
+                  <ListItem
+                    onPress={() =>
+                      handleDeleteItemPress(messageSelected._id as string)
+                    }
+                    style={{ paddingLeft: insets.left }}
+                    containerStyle={styles.messageOptionsContainer}
+                    testID={CHAT_MESSAGE_OPTIONS_DELETE}
+                  >
+                    <MaterialIcons name="delete" size={24} color="grey" />
+                    <ListItem.Content style={{ flexDirection: 'row' }}>
+                      <ListItem.Title style={{ flex: 2 }}>
+                        {t('Delete')}
+                      </ListItem.Title>
+                    </ListItem.Content>
+                  </ListItem>
+                </>
+              )}
+            </View>
+          </NativeViewGestureHandler>
+        </BottomSheetModal>
+      </>
+
+      <View style={{ ...styles.container, paddingBottom: insets.bottom }}>
+        <GiftedChat
+          {...{ messages, onSend }}
+          user={{
+            _id: currentMember?.id,
+          }}
+          text={inputMessage}
+          onInputTextChanged={(message) => setInputMessage(message)}
+          inverted={false}
+          alignTop={false}
+          renderBubble={renderBubble}
+          renderComposer={renderComposer}
+          renderMessageText={renderMessageText}
+        />
+      </View>
+    </>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+  },
+  messageContainer: {
+    width: '100%',
+    flexDirection: 'row',
+  },
+  bottomSheetModal: {
+    shadowColor: '#000',
+    shadowOffset: {
+      width: 0,
+      height: 12,
+    },
+    shadowOpacity: 0.58,
+    shadowRadius: 16.0,
+    elevation: 24,
+  },
+  composerContainer: {
+    width: '100%',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingTop: 5,
+  },
+  inputContainer: {
+    flex: 1,
+    backgroundColor: 'white',
+  },
+  suggestionContainer: {
+    position: 'absolute',
+    bottom: 30,
```

Comment:
This `absolute` style destroys the layout when you have a multiline message. It just appears on top of you message. I would prefer to have it appear above the message area.